client/blocks/inline-help/contextual-help.js

```diff
@@ -733,18 +750,21 @@ const contextLinksForSection = {
 		},
 		{
 			link: 'https://en.support.wordpress.com/',
+			post_id: 20158,
```

Comment:
Something seems wrong with this one. I'm not sure we would have a `post_id` for this link as it should link out to the support homepage rather than an individual post, right?

Here's what I see when clicking the link from inline-help, at the `/help` route:

<img width="1680" alt="screenshot 2019-01-23 at 13 12 44" src="https://user-images.githubusercontent.com/4335450/51609130-35db9980-1f11-11e9-8fe2-3b1316b2f234.png">

Reply:
Tracking here: https://github.com/Automattic/wp-calypso/issues/30357